hello.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
remove this file, hope your going to follow proper folder structure once u push your code